themes/base.theme.bash

```diff
@@ -98,6 +98,44 @@ function scm_prompt_info {
   [[ $SCM == $SCM_SVN ]] && svn_prompt_info && return
 }
 
+function scm_prompt_status {
```

Comment:
Also, wouldn't it be smarter to extend the existing `scm_prompt_info` function instead? Most of that function's code is duplicated here.

I suggest you introduce an environment variable like `SCM_GIT_SHORT_STATUS`, and if that is set, you call your new `git_prompt_status` function instead of the existing one. If the variable is not set, the current behavior will be used. This way, we avoid code duplication, preserve backwards compatibility and make it possible to switch functionality without having to rewrite themes.